src/utils/date-utils-test.js

```diff
@@ -57,7 +57,7 @@ describe('convertPotentialTimestampToSeconds', () => {
   });
   it('converts microseconds to seconds', () => {
     const nowInMicroseconds = Date.now() * 1000;
-    const nowInSeconds = Math.floor(nowInMicroseconds / 100000);
+    const nowInSeconds = Math.floor(nowInMicroseconds / 1000000);
```

Comment:
Sorry, also please rewrite this test so it is not duplicating the implementation.

Reply:
Did you mean the `const nowInMicroseconds = Date.now() * 1000;` is duplicated with `const nowInMilliseconds = Date.now();` for instance?
We can have literal values for second/milliseconds/microsecond to finish the tests?

Reply:
It's the same issue as the one mentioned here: https://github.com/subscriptions-project/swg-js/pull/2415#pullrequestreview-1157110290. The test should not duplicate the implementation logic, since then the test will pass even if the implementation is faulty (as happened when this bug was introduced).

Reply:
Thanks, let me know if the new commit addressed your concern.

Reply:
Yes, thanks!